src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -110,17 +106,17 @@
   }
 
   public List<Either<Command, CodeAction>> computeCodeActions(CodeActionParams params, CancelChecker cancelToken) {
-    List<Either<Command, CodeAction>> codeActions = new ArrayList<>();
-    URI uri = create(params.getTextDocument().getUri());
-    Optional<ProjectBindingWrapper> binding = bindingManager.getBinding(uri);
-    for (Diagnostic d : params.getContext().getDiagnostics()) {
+    var codeActions = new ArrayList<Either<Command, CodeAction>>();
+    var uri = create(params.getTextDocument().getUri());
+    var binding = bindingManager.getBinding(uri);
+    for (var d : params.getContext().getDiagnostics()) {
```

Comment:
With your change, it make it even clear that the variable name should be something more descriptive, like `diag` or `diagnostic`